src/main/java/com/prgrms/springbootbasic/handler/menu/MenuHandler.java

```diff
@@ -0,0 +1,49 @@
+package com.prgrms.springbootbasic.handler.menu;
+
+import com.prgrms.springbootbasic.console.Console;
+import com.prgrms.springbootbasic.handler.menu.dto.MenuInputResult;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MenuHandler {
+    private static final String COMMAND_NOT_SUPPORTER = "command not supported yet.";
+
+    private final Console console;
+
+    public MenuHandler(Console console) {
+        this.console = console;
+    }
+
+    public boolean handleBeforeExit() {
+        MenuInputResult inputResult = console.getCommand();
+        try {
+            CommandType commandType = CommandType.findByCommand(inputResult.getCommand());
+            handleMenu(commandType);
+            return commandType.isExit();
+        } catch (IllegalArgumentException e) {
+            console.printInvalidMessage(e.getMessage());
+            return false;
+        }
+    }
+
+    private void handleMenu(CommandType command) {
+        switch (command) {
+            case EXIT -> exit();
+            case CREATE -> create();
+            case LIST -> list();
+            default -> console.printInvalidMessage(COMMAND_NOT_SUPPORTER);
```

Comment:
잘못 된 메시지기 무엇인지 알려줄지 알았는데, 에러 메시지를 알려주고있네요?

전자를 생각하셨다면 아래처럼, 
```suggestion
            default -> console.printInvalidMessage(command);
```

후자라면 출력객체가 책임지는게 좋아보입니다. 
```suggestion
            default -> console.printInvalidMessage();
```

Reply:
@hongbin-dev 출력객체에서 COMMAND_NOT_SUPPORTED 문자열을 전달해주고, 출력객체가 해당 문자열을 출력하려는 의도였는데, 출력객체가 COMMAND_NOT_SUPPORTED를 갖고있는게 더 나은 방식인가요?

Reply:
넵 출력 내용이 바뀌면 출력객체가 변경되어야하는데요. `MenuHandler`를 변경해야하는 단점이 있네요.
출력객체의 책임이라고 생각됩니다.